applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app.overlay

```diff
@@ -68,3 +68,13 @@
 &cpusec_bellboard {
 	status = "okay";
 };
+
+/* Remove ppr partition as ppr core is not used and extend app core code partition. */
+/delete-node/ &cpuppr_code_partition;
```

Comment:
Consider setting partition size to zero instead of deleting node (if any node refers to `cpuppr_code_partition`, it will need to be manually aligned too). As an alternative, we could reduce PPR partition size (e.g. cut in half).